docs/manual/source/scipy-signal.rst

```diff
@@ -2,9 +2,10 @@
 scipy.signal
```

Comment:
The documentation is actually generated from this: https://github.com/v923z/micropython-ulab/blob/master/docs/scipy-signal.ipynb. The jupyter notebook allows you to run `micropython` code directly, so you don't have to copy anything.

Reply:
Ha ok, I made the changes in the jupyter notebook but I'm not sure how to run it (my system is windows).